File: src/main/java-graphql/io/github/graphql/java/model/SecurityAdvisoryQueryResponse.java
package io.github.graphql.java.model;

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult;
import java.util.Map;

@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-07T19:46:14+0800"
)
public class SecurityAdvisoryQueryResponse extends GraphQLResult<Map<String, SecurityAdvisoryTO>> {

    private static final String OPERATION_NAME = "securityAdvisory";

    public SecurityAdvisoryQueryResponse() {
    }

    public SecurityAdvisoryTO securityAdvisory() {
        Map<String, SecurityAdvisoryTO> data = getData();
        return data != null ? data.get(OPERATION_NAME) : null;
    }

}

File: src/main/scala-graphql/io/github/graphql/scala/model/SecurityAdvisoryQueryResponse.scala
package io.github.graphql.scala.model

import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResult
import java.util.{ Map => JMap }

@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-07T19:49:37+0800"
)
class SecurityAdvisoryQueryResponse extends GraphQLResult[JMap[String, SecurityAdvisoryTO]] {

    def securityAdvisory(): SecurityAdvisoryTO = {
        val data: JMap[String, SecurityAdvisoryTO] = getData
        if (data != null) data.get(SecurityAdvisoryQueryResponse.OPERATION_NAME) else null
    }

}

object SecurityAdvisoryQueryResponse {

    private final val OPERATION_NAME: String = "securityAdvisory"

}
